src/main/java/org/jenkinsci/plugins/workflow/support/steps/ExecutorStepExecution.java

```diff
@@ -535,6 +536,10 @@ public String getShortDescription() {
             return hasPermission(Item.CANCEL);
         }
 
+        /**
+         * @deprecated use {@link #getOwnerExecutable} (which does not require a dependency on this plugin).
```

Comment:
Potentially confusing without a core bump, as a dependency on this plugin or core ~2.389 is needed.

Reply:
True. Will keep this in draft unless & until the core change is merged and I can add a version number.